static/bidder-params/adtrgtme.json

```diff
@@ -4,5 +4,12 @@
   "description": "A schema which validates params accepted by the Adtrgtme adapter",
   "type": "object",
 
-  "properties": {}
+  "properties": {
+    "site_id": {
+      "type": "integer",
+      "description": "Site ID"
+    }
+  },
+
+  "required": ["site_id"]
```

Comment:
Adding a required parameter may cause the request tot fail. As per the design,  any request that has any bidder that fails the JSON schema will invalidate the entire request. The idea was that this would only happen when a publisher creates/updates their prebid config. When they do this, they should be looking to see if prebid is returning any bids, but probably aren't sensitive to if all bidders are bidding. They would otherwise not notice that a bidder is misconfigured until at some point they look at their analytics and notice the bidder is not bidding on a particular page or configuration setup. We think the current design choice is the better option to ensure everyone has a chance to bid. 
The recommended way here is to add `site_id` as optional and then have the adapter check and return an error if `site_id` is missing. If the parameters fail the schema check, we fail the entire request. If an adapter returns an error, we allow the other adapters to keep going and don't fail the whole request. At a later point in time, when the publishers have all had a chance to update to the new schema, we can make it required here. We just don't want all the publisher who are currently using your adapter to suddenly have all their prebid configs invalidated by surprise.

Reply:
Hello!
A very long description, but it still needs to be clarified why we can't use required fields as opposed to other adapters.

Reply:
If there is no traffic to this adapter then it is safe to add a required field.

Reply:
@grakholsky Thank you for being patient on this one!
The suggestion was if adapter suddenly requires site_id to work then the integration with the current publishers might fail as they might need some time to add this required field in their request. Now since you have clarified that there is no traffic to this adapter at the moment then this PR is good to go in that perspective. Hope this helps!